packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -396,6 +396,8 @@ class ScheduleManagerCore {
         for (const pending of allPending) {
             this.trackPending(pending);
         }
+
+        this.deltaManager.on("op", (message) => this.afterOpProcessing(message.sequenceNumber));
```

Comment:
Maybe add a comment that we are intentionally directly listening to the "op" to inspect system ops as well.
```suggestion
        // The ScheduleManager wants to inspect all ops (including system ops) and pause the queue just before
        // processing the start of a batch if we don't have the complete batch yet.
        this.deltaManager.on("op", (message) => this.afterOpProcessing(message.sequenceNumber));
```